ql/src/test/queries/clientpositive/create_table_like_file.q

```diff
@@ -0,0 +1,80 @@
+--! qt:dataset:src
```

Comment:
Do you reference the `src` table from any of the statements in this q file? If not please remove this line.

Reply:
HIVE-26398